sdk/src/main/java/com/bugsnag/android/Bugsnag.java

```diff
@@ -326,12 +326,14 @@ public static void setSessionTrackingApiClient(@NonNull SessionTrackingApiClient
 
 
     /**
-     * Add a "before notify" callback, to execute code before every
-     * report to Bugsnag.
+     * Add a "before notify" callback, to execute code before sending
+     * non-fatal reports to Bugsnag.
```

Comment:
C/C++ _and_ killed the app immediately. C `notify()` uses beforeNotify callbacks.

Reply:
In the paragraph below this line, the doc comment goes into more detail stating:

> "Before notify" callbacks do not run before reports generated in the event of immediate app termination, like crashes with C/C++ code.

Reply:
My reading of this is that `beforeNotify` is not invoked on Throwables caught by the `UncaughtExceptionHandler`, which is false. I think we should reword to avoid giving that impression.

Reply:
I agree with @fractalwrench  on this one - that could be clearer.